python/lsst/analysis/tools/atools/deltaSkyCorr.py

```diff
@@ -0,0 +1,67 @@
+# This file is part of analysis_tools.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+__all__ = ("DeltaSkyCorrXYPlot",)
+
+from ..actions.plot.xyPlot import XYPlot
+from ..actions.scalar.scalarActions import IqrHistAction, MedianHistAction
+from ..actions.vector import ConstantValue, LoadVector
+from ..interfaces import AnalysisTool
+
+
+class DeltaSkyCorrXYPlot(AnalysisTool):
+    parametrizeBands = False
```

Comment:
Do we really not want this to be done in every band?

Reply:
These are visit-level plot/metrics, so 1 visit = 1 band (i.e., there aren't multiple bands available).

Reply:
Discussed in an in-person chat, and agreed out of scope for this ticket.